Project.toml

```diff
@@ -11,6 +11,18 @@ RecipesBase = "3cdcf5f2-1ef4-517c-9805-6587b60abb01"
 Requires = "ae029012-a4dd-5104-9daa-d747884805df"
 Statistics = "10745b16-79ce-11e8-11f9-7d13ad32a3b2"
 
+[weakdeps]
+Juno = "e5e0dc1b-0480-54bc-9374-aad01c23163d"
+RecipesBase = "3cdcf5f2-1ef4-517c-9805-6587b60abb01"
+SpecialFunctions = "276daf66-3868-5448-9aa4-cd146d93841b"
+Unitful = "1986cc42-f94f-5a68-af5c-568840ba703d"
+
+[extensions]
+MeasurementsJunoExt = "Juno"
+MeasurementsSpecialFunctionsExt = "SpecialFunctions"
+MeasurementsUnitfulExt = "Unitful"
+MeasurementsRecipesBaseExt = "Unitful"
```

Comment:
```suggestion
MeasurementsRecipesBaseExt = "RecipesBase"
```
?

Reply:
that is the error i was trying to find hahah